packages/react-components/src/components/Accordion/Accordion.module.scss

```diff
@@ -0,0 +1,88 @@
+$base-class: 'accordion';
+
+.#{$base-class} {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: space-between;
+  transition: all var(--transition-duration-moderate-1);
+  border: 1px solid transparent;
+  border-radius: var(--radius-4);
+  box-shadow: unset;
+  background-color: var(--surface-secondary-default);
+  width: 100%;
+  min-height: 24px;
+
+  &:focus-visible {
+    outline: 0;
+    box-shadow: var(--shadow-focus);
+  }
+
+  &:hover {
+    border-color: var(--border-basic-hover);
+  }
+
+  &--warning {
+    background-color: var(--surface-accent-emphasis-min-warning);
+
+    &:hover {
+      border-color: var(--border-basic-warning);
+    }
+  }
+
+  &--error {
+    background-color: var(--surface-accent-emphasis-min-negative);
+
+    &:hover {
+      border-color: var(--content-basic-negative);
+    }
+  }
+
+  &--open {
+    box-shadow: var(--shadow-pop-over);
+    background-color: var(--surface-primary-default);
+
+    &:hover {
+      border-color: transparent;
+    }
+  }
+
+  &__chevron {
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    transition: inherit;
+    z-index: 0;
```

Comment:
do we need to set it to 0 here?

Reply:
Yes, it is used in animation to fade between open/close label variants.

Reply:
Yes, it makes the whole label clickable (which is expected) what is not covered by chevron icon.

Reply:
I will change that for `pointer-events`